From a36a41bf39306ea88a22fc3487809dcc41c8b27d Mon Sep 17 00:00:00 2001
From: Tony Xie <tony.xie@rock-chips.com>
Date: Tue, 6 Feb 2018 15:50:03 +0800
Subject: [PATCH] arm64: dts: rockchip: change the vdd_arm and vdd_logic node
 of px30/rk3326 board

change the vdd_arm and vdd_logic dts node for rk3326 and px30.
delete the pinctrl define for rk818 and rk809.

Change-Id: I4fcc10ac48b4b91f4225efc2efdb78227de005ba
Signed-off-by: Tony Xie <tony.xie@rock-chips.com>
---
 .../boot/dts/rockchip/px30-evb-ddr3-v10.dts   | 28 +++----------------
 .../boot/dts/rockchip/px30-evb-ddr4-v10.dts   | 28 +++----------------
 .../boot/dts/rockchip/rk3326-863-lp3-v10.dts  | 28 +++----------------
 .../boot/dts/rockchip/rk3326-evb-lp3-v10.dts  | 28 +++----------------
 4 files changed, 16 insertions(+), 96 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
index 199b8279660f..4e2cfb54d601 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
@@ -297,49 +297,29 @@
 		vcc8-supply = <&vcc3v3_sys>;
 		vcc9-supply = <&vcc5v0_sys>;
 
-		rk817_pin0_ts: rk817_pin0_ts {
-			pins = "gpio0";
-			function = "pin_fun0";
-		};
-
-		rk817_pin0_gpio: rk817_pin0_gpio {
-			pins = "gpio0";
-			function = "gpio";
-		};
-
-		rk817_pin1_gt: rk817_pin1_gt {
-			pins = "gpio1";
-			function = "pin_fun0";
-		};
-
-		rk817_pin1_gpio: rk817_pin1_gpio {
-			pins = "gpio1";
-			function = "gpio";
-		};
-
 		regulators {
-			vdd_arm: DCDC_REG1 {
+			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_arm";
+				regulator-name = "vdd_logic";
 				regulator-state-mem {
 					regulator-on-in-suspend;
 					regulator-suspend-microvolt = <700000>;
 				};
 			};
 
-			vdd_logic: DCDC_REG2 {
+			vdd_arm: DCDC_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_logic";
+				regulator-name = "vdd_arm";
 				regulator-state-mem {
 					regulator-off-in-suspend;
 					regulator-suspend-microvolt = <800000>;
diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
index 60cb556f9099..7ad17d7371eb 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
@@ -297,49 +297,29 @@
 		vcc8-supply = <&vcc3v3_sys>;
 		vcc9-supply = <&vcc5v0_sys>;
 
-		rk817_pin0_ts: rk817_pin0_ts {
-			pins = "gpio0";
-			function = "pin_fun0";
-		};
-
-		rk817_pin0_gpio: rk817_pin0_gpio {
-			pins = "gpio0";
-			function = "gpio";
-		};
-
-		rk817_pin1_gt: rk817_pin1_gt {
-			pins = "gpio1";
-			function = "pin_fun0";
-		};
-
-		rk817_pin1_gpio: rk817_pin1_gpio {
-			pins = "gpio1";
-			function = "gpio";
-		};
-
 		regulators {
-			vdd_arm: DCDC_REG1 {
+			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_arm";
+				regulator-name = "vdd_logic";
 				regulator-state-mem {
 					regulator-on-in-suspend;
 					regulator-suspend-microvolt = <700000>;
 				};
 			};
 
-			vdd_logic: DCDC_REG2 {
+			vdd_arm: DCDC_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_logic";
+				regulator-name = "vdd_arm";
 				regulator-state-mem {
 					regulator-off-in-suspend;
 					regulator-suspend-microvolt = <800000>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
index 5c20808e43f4..1724570e54f8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
@@ -233,49 +233,29 @@
 		vcc8-supply = <&vccsys>;
 		vcc9-supply = <&dcdc_boost>;
 
-		rk817_pin0_ts: rk817_pin0_ts {
-			pins = "gpio0";
-			function = "pin_fun0";
-		};
-
-		rk817_pin0_gpio: rk817_pin0_gpio {
-			pins = "gpio0";
-			function = "gpio";
-		};
-
-		rk817_pin1_gt: rk817_pin1_gt {
-			pins = "gpio1";
-			function = "pin_fun0";
-		};
-
-		rk817_pin1_gpio: rk817_pin1_gpio {
-			pins = "gpio1";
-			function = "gpio";
-		};
-
 		regulators {
-			vdd_arm: DCDC_REG1 {
+			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_arm";
+				regulator-name = "vdd_logic";
 				regulator-state-mem {
 					regulator-on-in-suspend;
 					regulator-suspend-microvolt = <700000>;
 				};
 			};
 
-			vdd_logic: DCDC_REG2 {
+			vdd_arm: DCDC_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_logic";
+				regulator-name = "vdd_arm";
 				regulator-state-mem {
 					regulator-off-in-suspend;
 					regulator-suspend-microvolt = <800000>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
index ac85ad3d589d..3fd623be9c07 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
@@ -330,49 +330,29 @@
 		vcc8-supply = <&vccsys>;
 		vcc9-supply = <&dcdc_boost>;
 
-		rk817_pin0_ts: rk817_pin0_ts {
-			pins = "gpio0";
-			function = "pin_fun0";
-		};
-
-		rk817_pin0_gpio: rk817_pin0_gpio {
-			pins = "gpio0";
-			function = "gpio";
-		};
-
-		rk817_pin1_gt: rk817_pin1_gt {
-			pins = "gpio1";
-			function = "pin_fun0";
-		};
-
-		rk817_pin1_gpio: rk817_pin1_gpio {
-			pins = "gpio1";
-			function = "gpio";
-		};
-
 		regulators {
-			vdd_arm: DCDC_REG1 {
+			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_arm";
+				regulator-name = "vdd_logic";
 				regulator-state-mem {
 					regulator-on-in-suspend;
 					regulator-suspend-microvolt = <700000>;
 				};
 			};
 
-			vdd_logic: DCDC_REG2 {
+			vdd_arm: DCDC_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <950000>;
 				regulator-max-microvolt = <1350000>;
 				regulator-ramp-delay = <6001>;
 				regulator-initial-mode = <0x1>;
-				regulator-name = "vdd_logic";
+				regulator-name = "vdd_arm";
 				regulator-state-mem {
 					regulator-off-in-suspend;
 					regulator-suspend-microvolt = <800000>;
-- 
2.35.3

